CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(backtrace)
+
+# Automake uses -frandom-seed initialized with file name of given file
+# but AFAIK it can't be done on CMake, so here's always same seed

```

Comment:
This should be possible, which file is expected to use as seed ?

Reply:
According to automake each file was using it's own file for seed. I don't think there's strong reason for that, so I guess we can just use same seed for every file.